.github/workflows/build-and-publish.yaml

```diff
@@ -117,7 +117,9 @@ jobs:
         with:
           go-version: '1.18'
       - name: Run tests
-        run: PATH=$PWD/bin:$PATH make test
+        run: |
+          go mod vendor
```

Comment:
Is that mean, we have the wrong version of dependencies in the branch? Otherwise, this command has no output. On the other hand, we test something which isn't reproducible because it always depends on current execution. WDYT?

Reply:
It seems the issue has resolved itself somehow 🧙 
You can see here attempt 1 failed, attempt 2 all good 🤷 https://github.com/storageos/operator/actions/runs/3656668894/jobs/6180817476

I wasn't able to reproduce it locally either